### ISSUE TITLE
Loss output

### ISSUE THREAD

**Issue by kobybibas** [akamaster/pytorch_resnet_cifar10#2]
Hi,
can you also report the loss you got on the trained model?
Thanks a lot

**Comment by akamaster** [comment]
I'll try to add it tonight, but cant guarantee. Anyway, you can always load pretrained model calculate loss on dataset if in a rush.

**Comment by Ontheroad123** [comment]
could please tell me the 'pretrained' is only weight or include the structure of model?
